testsuite/unittests/test_gif_pop_psc_exp.sli

```diff
@@ -38,6 +38,9 @@ SeeAlso: gif_pop_psc_exp, testsuite::test_pp_pop_psc_delta
 
 */
 
+% This test should only run if we have GSL
+statusdict/have_gsl :: not {statusdict/exitcodes/success :: quit_i} if
```

Comment:
Please move this check to after `(unittest) run` and replace it with `skip_if_without_gsl`. This is not only simpler, but it will also mark the test as skipped due to lack of GSL instead of implicitly counting it as passed.